ui/raidboss/data/07-dt/alliance/jeuno-first-walk.txt

```diff
@@ -0,0 +1,750 @@
+# 9FF3 Banish: Pathing circle AoEs
+# 9FF4 Crystalline Thorns: Thorn generation cast
+# 9FF5 Thornbite: Thorns appear
+# 9FF6 Auroral Uppercut: 1-tick knockback visual
+# 9FF7 Auroral Uppercut: 2-tick knockback visual
+# 9FF8 Auroral Uppercut: 3-tick knockback visual
+# 9FF9 Auroral Uppercut: Knockback execution
+# 9FFA Banishga IV: Raidwide; summons orbs
+# 9FFB Explosion: Orb chariot AoEs
+# 9FFC Asuran Fists: Stack marker cast
+# 9FFD Nullifying Dropkick: Shared tank buster, circle stack (simultaneous with 9FF1)
+# 9FFE Asuran Fists: Stack marker damage
+# 9FFF Asuran Fists: Stack marker damage
+# A000 Asuran Fists: Stack marker final hit
+# A002 Holy: Spread marker cast
+# A003 Holy: Spread marker damage
+
+
+#~~~~~~~~~~~~~~~~~~~~~#
+# Fafnir The Forgotten#
+#~~~~~~~~~~~~~~~~~~~~~#
+
+# -ii 9BF2 9EB9 9F72 9F73 9F74 9F75 9F76 9F8B 9F8C 9F95 9F98 9F9C A156 A157 A158 A159 9F90 9F8E 9F6F
+# -ic "Biting Wind" "Darter" "Ravaging Wind"
+
+# The Hurricane Wing puddles follow an increasing "ladder" of abilities.
+# In -> out first round:: 9F7D, 9F7E, 9F7F, 9F80
+# In -> out second round: 9F87, 9F88, 9F89, 9F8A
+# Out -> in first round:  9F78, 9F79, 9F7A, 9F7B
+# Out -> in second round: 9F82, 9F83, 9F84, 9F85
+
+# Dragon's Aery will be sealed off
+20000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "141F" } window 20000,1
+20011.1 "Dark Matter Blast" Ability { id: "9F96", source: "Fafnir the Forgotten" } window 20011.1,10
+20024.2 "Offensive Posture" Ability { id: ["9F6B","9F6E", "9F70"], source: "Fafnir the Forgotten" }
+20025.2 "Spike Flail" Ability { id: "A09A", source: "Fafnir the Forgotten" }
+20038.3 "Offensive Posture?" Ability { id: ["9F6B","9F6E", "9F70"], source: "Fafnir the Forgotten" }
+20039.5 "Touchdown?" Ability { id: "A09C", source: "Fafnir the Forgotten" }
+
+20044.5 "--center--" Ability { id: "9F99", source: "Fafnir the Forgotten" } # Shuddering Earth window 44.5,10
+20056.6 "Offensive Posture" Ability { id: ["9F6B","9F6E", "9F70"], source: "Fafnir the Forgotten" }
+20057.8 "Dragon Breath" duration 4 #Ability { id: "9F6F", source: "Fafnir the Forgotten" }
+20075.1 "Offensive Posture" Ability { id: ["9F6B","9F6E", "9F70"], source: "Fafnir the Forgotten" }
+20076.3 "Touchdown" Ability { id: "A09C", source: "Fafnir the Forgotten" }
+20087.3 "Offensive Posture" Ability { id: ["9F6B","9F6E", "9F70"], source: "Fafnir the Forgotten" }
+20088.3 "Spike Flail" Ability { id: "A09A", source: "Fafnir the Forgotten" }
+20098.7 "Baleful Breath x5" Ability { id: "9F94", source: "Fafnir the Forgotten" } duration 7
+20105.5 "--sync--" Ability { id: "9B76", source: "Fafnir the Forgotten" } # Baleful Breath final hit
+20112.6 "Sharp Spike" Ability { id: "9F97", source: "Fafnir the Forgotten" }
+20114.0 "--Darters spawn--"
+20123.7 "--center--" Ability { id: "9F99", source: "Fafnir the Forgotten" } # Shuddering Earth
+20130.8 "Hurricane Wing (raidwides)" Ability { id: "9F71", source: "Fafnir the Forgotten" } duration 11
```

Comment:
This repeating aoe could use a callout, it's a fair bit of damage.

Reply:
We also usually put the count of hits in the entry, so maybe it should be `Hurricane Wing x3` or something...

Reply:
We don't currently have a "repeated raidwides" canned response. Should we go with `Responses.aoe()` here, or maybe add the new canned response, or just settle for `AoE x10` in this one situation?

Good point on the timeline entry, I will change that regardless.